docs/docs/installation/installing-rasa-open-source.mdx

```diff
@@ -89,6 +84,25 @@ to install all needed dependencies for every configuration.
 
 :::
 
+*If you are using macOS*, note that installing `rasa[full]` (either via pip or from source) could result in a failure
+while installing `tokenizers` (issue described in depth [here](https://github.com/huggingface/tokenizers/issues/1050)).
+
+In order to resolve it, you must follow these steps to install a Rust compiler:
+```bash
+brew install rustup
```

Comment:
maybe worth adding to the list of [ignored words](https://github.com/RasaHQ/rasa/blob/89671f0847009f6c2d34c09a919b53e6219b11fc/.typo-ci.yml#L64)

Reply:
Addressed: https://github.com/RasaHQ/rasa/pull/11773/commits/bfd5135e0d7127da726a92b50602c29ae1991636